pandas/io/parsers.py

```diff
@@ -1541,11 +1532,9 @@ def _maybe_dedup_names(self, names):
                     counts[col] = cur_count + 1
 
                     if is_potential_mi:
-                        col = col[:-1] + (
-                            "{column}.{count}".format(column=col[-1], count=cur_count),
-                        )
+                        col = col[:-1] + (f"{col[-1]}.{cur_count}")
```

Comment:
```suggestion
                        col = col[:-1] + f"{col[-1]}.{cur_count}"
```
the parenthesis seem unnecesary